asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py

```diff
@@ -392,6 +388,13 @@ def _generate_pose(
             return_code != oeomega.OEOmegaReturnCode_Success
         ):
             # add the failure message as an SD tag, should be able to see visually if the molecule is 2D
+
+            # because of downstream inconsistencies in the way we handle SD data, here we'll just add both to the
+            # top level and conformations. There are testing errors if we don't do both.
```

Comment:
I think this should only need to be set on the molecule SD tag.

Reply:
I agree that it *should* be the case, but I get errors in `test_pose_generation` if I don't do both. I *think* it's because if you make a GraphMol from an OEMol it will lose conformation-specific SD data?